README.RDAP.md

```diff
@@ -45,21 +45,22 @@ For example: https://rdap.db.ripe.net/entity/KR4422-RIPE
 Related Contact information is Filtered
 ---------------------------------------
 All contact entities ("technical","administrative" etc.) have filtered contact information, i.e. "e-mail"
-and "notify" values are not included. Contact entities do not count towards the daily query limit
+value is not included. Contact entities do not count towards the daily query limit
```

Comment:
Clarify that *no* e-mail is included, "i.e. no e-mail addresses are included." (to include *all* attributes containing an email address, don't specify which attributes as there are quite a few of them)

Also replace the term "Contact entities" with "Related entities" to be consistent.

Reply:
This is already explained in the last paragraph. I think that it is redundant to talk about the request that have e-mail filtered.
I am ok with the rest of the comments.